docs/modules/ROOT/pages/cdc-cassandra-events.adoc

```diff
@@ -79,6 +80,8 @@ And a JSON representation of the AVRO Record for the message payload:
 
 When deleting a row, the Cassandra source connector will publish a tombstone message into the clean topic with a message key matching the C* primary key and a null payload. The null payload acts as a message tombstone, and the downstream Pulsar connectors like the Elasticsearch sink should delete the corresponding row (or document in the case of Elasticsearch).
 
+If using the `key-value-json` output format, the Pulsar message will pass an empty JSON instead of a null payload, because Pulsar doesn't support nulls in message values. The downstream connectors will still delete the corresponding row. 
```

Comment:
this is actually true in the case of `json` output format. `key-value-json` will behave exactly as the default case (which is `key-value-avro`) in which case a nulls are allowed.